### ISSUE TITLE
Merge with Symbolics.jl ?

### ISSUE THREAD

**Issue by maucejo** [SciML/SymbolicNumericIntegration.jl#75]
Hi all,

Is there any plan to include SymbolicIntegration.jl in Symbolics.jl ?

At the moment, I use SymPy.jl, because it is more complete CAS.

Thank you :)

**Comment by ChrisRackauckas** [comment]
Not really. It would be nice to have this as part of a polyalgorithm with a normal symbolic integration method, but it's such an abnormal method that it would be good to instead make sure we have a standard way of doing it and the propose an async mix method.

**Comment by maucejo** [comment]
Thanks Chris for your detailed answer (as usual).
